packages/components/src/ButtonDismiss/ButtonDismiss.mdx

```diff
@@ -0,0 +1,52 @@
+---
+name: ButtonDismiss
+menu: Components
+route: /components/button-dismiss
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { ButtonDismiss } from ".";
+
+# Button Dismiss
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+Write a description about what ButtonDismiss component is trying to achieve.
```

Comment:
please `Write a description about what ButtonDismiss component is trying to achieve.`

Reply:
This is now addressed in be4d592. Thanks!